en/docs/deploy-and-publish/deploy-on-gateway/choreo-connect/endpoints/overriding-endpoints-for-imported-apis.md

```diff
@@ -17,11 +17,11 @@ You can define endpoint details as the Adapter environment variables at docker-c
     services:
         adapter:
             environment:
-            - api_60f7111f-fdc5-4cc7-b497-1cea64c6a97f_prod_endpoint_0="http://wso2.prod.com"
-            - api_60f7111f-fdc5-4cc7-b497-1cea64c6a97f_sand_endpoint_0="http://wso2.sand.com"
+            - api_cc429c240db698dd2522038f8f85602bfcd33ed3_prod_endpoint_0="http://wso2.prod.com"
+            - api_cc429c240db698dd2522038f8f85602bfcd33ed3_sand_endpoint_0="http://wso2.sand.com"
     ```
     
-    - `<API-ID>` - `api_SHA1(<apiName>:<apiVersion>)` ID generated for the imported API. Sample name would be `api_30e623704c5c5479b7c0d9ab78e965df02c1610401e37cbd557e6353e3191c76`.
+    - `<API-ID>` - `api_SHA1(<apiName>:<apiVersion>)` ID generated for the imported API. Sample name would be `api_cc429c240db698dd2522038f8f85602bfcd33ed3`.
```

Comment:
Shall we mention how user can generate this sha1? for example `echo -n "Petstore:1.0.0" | openssl sha1`

Reply:
Fixed in [faaa0cc](https://github.com/wso2/docs-apim/pull/7411/commits/faaa0ccdb67c17f386694a60e6b0de36b2a28244)

<img width="1004" alt="Screenshot 2023-11-17 at 14 47 11" src="https://github.com/wso2/docs-apim/assets/62060834/a232635f-9535-472a-80be-f0e80d4221d2">